portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,62 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Gochi+Hand&family=Manrope&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <header id="intro">
+            <h3>Hi I'm</h3>
+            <h2 id="name"> Rebecca Sun</h2>
+        </header>
+
+        <h4 id="categories">-ABOUT-</h4>
```

Comment:
The "id" attribute is meant to uniquely identify one single element on the page.  i.e. if we set id="categories" on this h4, we wouldn't want to write id="categories" anywhere else on the page.  I'm not actually sure what kind of errors will happen when this rule gets broken.. haha.. but best to follow the rules of the web!

When you want to define some CSS styling that can be applied to multiple different elements like in your case, it is best to use a class instead of an ID.  I like the pretty short and to the point explanation they give here on IDs vs classes: https://www.codecademy.com/articles/classes-vs-ids

Also, naming wise, I tend to see classes (and IDs) being named in the singular instead of the plural.  I don't know if that's an explicit rule, but I think it's a bit more normal/readable.  Plus, people reading your code will know that classes can apply to multiple elements, so the plurality is implied without you having to tack on an 's' to the end of all your classes =)

So in this case I'd say go with:
<h4 class="category">-ABOUT-</h4>

...and then the same changes with the other h4s where you are currently using an id.

Reply:
Oh no haha... github changed my HTML example into bolded text.  Basically I was just saying go with class="category" instead of id="categories" and just don't have an ID on the h4 at all =)